orcestra/flightplan.py

```diff
@@ -485,6 +485,46 @@ def is_entry_through(self):
             return False
         return (self.enter + 180) % 360 - 180 > 0
 
+    def get_intersect(self, other: IntoCircle):
+        fwd, _, distance = geod.inv(
+            self.center.lon, self.center.lat, other.center.lon, other.center.lat
+        )
+        if distance > self.radius + other.radius:
+            raise ValueError(
+                "The two circles are too far apart. There is no intersection"
+            )
+        else:
+
+            def cost(fwd_az):
+                fwd_az = (fwd_az + 180) % 360 - 180
+                # print(fwd_az)
+                fwd_res = (
+                    geod.fwd(self.center.lon, self.center.lat, fwd_az, self.radius),
+                )
+                _, _, dist = geod.inv(
+                    fwd_res[0][0], fwd_res[0][1], other.center.lon, other.center.lat
+                )
+                return np.abs(dist - other.radius)
```

Comment:
Not sure if that actually improves things, but somewhere back in my math classes I've learned that quadratic cost functions tend to have better convergence. If that's true, then we might want something like:

```suggestion
                return (dist - other.radius)**2
```

Reply:
I changed this  - in order for it to work with the square, I had to change to the default minimize method, but that's no problem I guess.